autoware.repos

```diff
@@ -38,7 +38,7 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware.universe.git
     version: main
-  universe/external/tier4_ad_api_adaptor: # TODO(TIER IV): Migrate to AD API
+  universe/external/tier4_ad_api_adaptor:
```

Comment:
I think this was about removing the `tier4_ad_api_adaptor` altogether by migrating to AD API.

It is expected to remove this repository reference from the .repos file to complete that TODO item.

Reply:
```suggestion
  universe/external/tier4_ad_api_adaptor: # TODO(TIER IV): Migrate to AD API and remove this repository entry.
```

This would avoid confusion in the future.

Reply:
I think you have resolved it but not applied the change?

![image](https://github.com/user-attachments/assets/a1477978-9e52-4ba8-aa13-15cc1571d34b)

Reply:
I'm sorry I do it again.

Reply:
No worries, thanks <3